raiden/encoding/messages.py

```diff
@@ -257,7 +242,7 @@ def wrap_and_validate(data):
 
     try:
         publickey = recover_publickey(message_data, message_signature)
-    except Exception:
+    except:  # pylint: disable=bare-except
```

Comment:
`recover_publickey()` itself raises only a `ValueError`. 

But the `secp256k1` functions that are also invoked inside there throw simple `Exception`.

Reference: https://github.com/ludbb/secp256k1-py/blob/master/secp256k1/__init__.py#L132